src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -229,6 +217,12 @@ protected CreatureStageHUDBase()
     [Signal]
     public delegate void OnOpenMenuToHelpEventHandler();
 
+    [Signal]
+    public delegate void OnSetEditorButtonFlashEffectEventHandler();
```

Comment:
Instead of a signal here, this should just directly call methods on the editor button. Note that I see some variables like `protected TextureButton editorButton = null!;` still existing. That should be updated to be:

```c#
[Export]
protected EditorEntryButton editorButton = null!;
```

To be able to call the C# methods on that variable.